Client/src/Modelo/Torneo.java

```diff
@@ -0,0 +1,62 @@
+package Modelo;
+
+import java.util.ArrayList;
+
+/**
+ * @author Grupo
+ * 
+ *         Esta clase representa el funcionamiento de un torneo eliminatorio
+ *         para 8 competidores. Cuenta con el conjunto de entrenadores que
+ *         participaran en dicho torneo, el nombre del torneo y una
+ *         variable que hara referencia a una sola instancia del
+ *         torneo.
+ */
+public class Torneo
+{
+	private String nombre_torneo;
```

Comment:
```suggestion
	private String nombre;
```
Para respetar la nomenclatura determinada en Java. Ademas si ya es de la clase Torneo, volver a hacer referencia a que el nombre es del torneo es irrelevante